app/views/locks/_button.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if can? :write, :locks, @stage %>
+<% if can? :write, :locks, resource %>
```

Comment:
make sure to click through environment/deploy-group/project and check that it still works, idk why `@stage` was here, but now the logic will be a lot different ...